components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -894,8 +936,124 @@ subroutine aero_model_init( pbuf2d, species_class, iflagaa )
+    index_chm_mass(2,2) = get_spc_ndx('nh4_a2')
+    index_chm_mass(2,3) = get_spc_ndx('soa_a2')
+    !
+    ! coarse mode
+    !
+    index_tot_mass(3,1)  = get_spc_ndx('dst_a3')
+    index_tot_mass(3,2)  = get_spc_ndx('ncl_a3')
+    index_tot_mass(3,3)  = get_spc_ndx('so4_a3')
+    index_tot_mass(3,4)  = get_spc_ndx('bc_a3')
+    index_tot_mass(4,5)  = get_spc_ndx('pom_a3')
+    index_tot_mass(4,6)  = get_spc_ndx('soa_a3')
+    index_tot_mass(4,7)  = get_spc_ndx('mom_a3')
+    index_tot_mass(3,8)  = get_spc_ndx('nh4_a3')
+    index_tot_mass(3,9)  = get_spc_ndx('no3_a3')
+    index_tot_mass(3,10) = get_spc_ndx('ca_a3')
+    index_tot_mass(3,11) = get_spc_ndx('co3_a3')
+    index_tot_mass(3,12) = get_spc_ndx('cl_a3')
+    index_chm_mass(3,1)  = get_spc_ndx('so4_a3')
+    index_chm_mass(3,2)  = get_spc_ndx('nh4_a3')
+    index_chm_mass(3,3)  = get_spc_ndx('soa_a3')
+    index_chm_mass(3,4)  = get_spc_ndx('bc_a3')
+    !
+    ! POM mode
+    !
+    index_tot_mass(4,1) = get_spc_ndx('pom_a4')
+    index_tot_mass(4,2) = get_spc_ndx('bc_a4')
+    index_tot_mass(4,3) = get_spc_ndx('mom_a4')
+    index_chm_mass(4,1) = get_spc_ndx('bc_a4' )
+    !
+    ! strat_coarse
+    index_tot_mass(5,1) = get_spc_ndx('so4_a5')
+    index_chm_mass(5,1) = get_spc_ndx('so4_a5')
+    !
+#elif ( ( defined MODAL_AERO_5MODE ) && ( defined RAIN_EVAP_TO_COARSE_AERO ) )
+    !
+    ! accumulation mode #1
+    !
+    index_tot_mass(1,1) = get_spc_ndx('so4_a1')
+    index_tot_mass(1,2) = get_spc_ndx('pom_a1')
+    index_tot_mass(1,3) = get_spc_ndx('soa_a1')
+    index_tot_mass(1,4) = get_spc_ndx('bc_a1' )
+    index_tot_mass(1,5) = get_spc_ndx('dst_a1')
+    index_tot_mass(1,6) = get_spc_ndx('ncl_a1')
+    index_tot_mass(1,7) = get_spc_ndx('mom_a1')
+    index_chm_mass(1,1) = get_spc_ndx('so4_a1')
+    index_chm_mass(1,2) = get_spc_ndx('soa_a1')
+    index_chm_mass(1,3) = get_spc_ndx('bc_a1' )
+    !
+    ! aitken mode
+    !
+    index_tot_mass(2,1) = get_spc_ndx('so4_a2')
+    index_tot_mass(2,2) = get_spc_ndx('soa_a2')
+    index_tot_mass(2,3) = get_spc_ndx('ncl_a2')
+    index_tot_mass(2,4) = get_spc_ndx('mom_a2')
+    index_chm_mass(2,1) = get_spc_ndx('so4_a2')
+    index_chm_mass(2,2) = get_spc_ndx('soa_a2')
+    !
+    ! coarse mode
+    !
+    index_tot_mass(3,1)  = get_spc_ndx('dst_a3')
+    index_tot_mass(3,2)  = get_spc_ndx('ncl_a3')
+    index_tot_mass(3,3)  = get_spc_ndx('so4_a3')
+    index_tot_mass(3,4)  = get_spc_ndx('bc_a3')
+    index_tot_mass(4,5)  = get_spc_ndx('pom_a3')
+    index_tot_mass(4,6)  = get_spc_ndx('soa_a3')
+    index_tot_mass(4,7)  = get_spc_ndx('mom_a3')
+    index_chm_mass(3,1)  = get_spc_ndx('so4_a3')
+    index_chm_mass(3,2)  = get_spc_ndx('soa_a3')
+    index_chm_mass(3,3)  = get_spc_ndx('bc_a3')
+    !
+    ! POM mode
+    !
+    index_tot_mass(4,1) = get_spc_ndx('pom_a4')
+    index_tot_mass(4,2) = get_spc_ndx('bc_a4')
+    index_tot_mass(4,3) = get_spc_ndx('mom_a4')
+    index_chm_mass(4,1) = get_spc_ndx('bc_a1' )
```

Comment:
Is "bc_a1" in this line a mistake? It is also in the default option two blocks above.

Reply:
@hwangacme  Yes. It is. Will correct it.